src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String TEST_LOGIN = "Kolya";
+    private static final String TEST_PASSWORD = "password";
+    private static final Integer TEST_AGE = 23;
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+      void registrationService_registerValidUser_ok() {
+        User user = new User();
+        User actual = user;
```

Comment:
```suggestion
        User actual = new User();
```

навіщо нам тут створювати 2 змінні User, які посилаються на 1 об'єкт?

Reply:
Таки да, не за чем. Убрал